.travis.yml

```diff
@@ -19,105 +19,57 @@ cache:
 matrix:
   fast_finish: true
   allow_failures:
-    - os: windows
-    - os: osx
+    - env: TEST_CONTAINER=latest
+    - env: TEST_CONTAINER=py37
```

Comment:
I think you need to list all envs here, e.g.:

```
    - env: TEST_CONTAINER=latest OGGM_TEST_ENV=utils MPL=
      os: linux
```

Reply:
What do you think about removing the latest tests instead?

Reply:
> What do you think about removing the latest tests instead?

Yes! Good idea.